k/kong/kong-3.3.0.patch

```diff
@@ -0,0 +1,412 @@
+diff --git a/Makefile b/Makefile
+index 917e0070b..7f231a5d3 100644
+--- a/Makefile
++++ b/Makefile
+@@ -39,7 +39,7 @@ endif
+ 
+ ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+ KONG_SOURCE_LOCATION ?= $(ROOT_DIR)
+-GRPCURL_VERSION ?= 1.8.5
++GRPCURL_VERSION ?= 1.8.7
+ BAZLISK_VERSION ?= 1.16.0
+ BAZEL := $(shell command -v bazel 2> /dev/null)
+ VENV = /dev/null # backward compatibility when no venv is built
+@@ -67,9 +67,7 @@ build-kong: check-bazel
+ build-venv: check-bazel
+ 	$(eval VENV := bazel-bin/build/$(BUILD_NAME)-venv.sh)
+ 
+-	@if [ ! -e bazel-bin/build/$(BUILD_NAME)-venv.sh ]; then \
+-		$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME); \
+-	fi
++	$(BAZEL) build //build:venv --verbose_failures --action_env=BUILD_NAME=$(BUILD_NAME)
+ 
+ install-dev-rocks: build-venv
+ 	@. $(VENV) ;\
+@@ -86,7 +84,6 @@ install-dev-rocks: build-venv
+ dev: build-venv install-dev-rocks bin/grpcurl
+ 
+ build-release: check-bazel
+-	$(BAZEL) build clean --expunge
```

Comment:
why are we removing this?

Reply:
expunge is no longer supported by bazel, removed from master as well